src/wl_window.c

```diff
@@ -208,6 +208,13 @@ static GLFWbool createSurface(_GLFWwindow* window,
     if (!window->wl.native)
         return GLFW_FALSE;
 
+    window->wl.idleInhibitor =
```

Comment:
This must only be done if _glfw.wl.idleInhibitManager is set, i.e. the compositor supports the protocol.